manifests/m/Mp3tag/Mp3tag/3.21/Mp3tag.Mp3tag.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.2.7 $debug=QUSU.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mp3tag.Mp3tag
+PackageVersion: "3.21"
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: Mp3tag v3.18
```

Comment:
```suggestion
```
Not needed as we can get reliable package matching with other fields in the manifest